src/main/java/racingcar/application/RacingGameManager.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.application;
+
+import racingcar.domain.RacingGame;
+import racingcar.domain.RacingGameRepository;
+import racingcar.domain.RacingGameResultsDto;
+
+public class RacingGameManager {
+
+    private final RacingGameRepository racingGameRepository;
+
+    public RacingGameManager(RacingGameRepository racingGameRepository) {
+        this.racingGameRepository = racingGameRepository;
+    }
+
+    public RacingGameResultsDto game(RacingGame racingGame) {
+        racingGameRepository.saveAll(racingGame.start());
```

Comment:
```suggestion
        List<Cars> result = racingGame.start();
        racingGameRepository.save(result);
```
지역변수를 두는 것이 때론 의미를 분명히 전달하는 경우도 있다고 생각합니다.